.gitignore

```diff
@@ -33,5 +33,7 @@ secrets.tfvars
 terraform.tfstate*
 *.tar
 *.tar.zip
-mddata/
-ncdata/
+mddata
+ncdata
```

Comment:
A bunch of other ignored directories also have trailing "/".  I'd add it back for consistency and to simplify the diff.